lib/chef/resource/windows_feature_powershell.rb

```diff
@@ -54,9 +54,19 @@ class WindowsFeaturePowershell < Chef::Resource
                description: "",
                default: false
 
-      def to_lowercase_array(x)
+      # a simple helper to determine if we're on a windows release pre-2012 / 8
+      # @return [Boolean] Is the system older than Windows 8 / 2012
+      def older_than_2012_or_8?
```

Comment:
Arguably this could go into chef/platform/query_helpers as `Chef::Platform.older_than_2012_or_8?` so we can reuse it.